src/metabase/automagic_dashboards/populate.clj

```diff
@@ -43,9 +44,22 @@
         :location "/")
       (create-collection! "Automatically Generated Dashboards" "#509EE3" nil nil)))
 
-(def colors
+(defn- get-whitelabeled-colors
+  "Get color from application-colors or default color."
+  [default-colors]
+  (let [whitelabeled-color-names [:brand :accent1 :accent2 :accent3 :accent4 :accent5 :accent6 :accent7]]
+    (->> default-colors
+         (map vector whitelabeled-color-names)
+         (map (fn [[whitelabeled-color-name default-color]]
+                (whitelabeled-color-name (public-settings/application-colors) default-color)))
+         (into []))))
+
+(defn colors
   "Colors used for coloring charts and collections."
-  ["#509EE3" "#9CC177" "#A989C5" "#EF8C8C" "#f9d45c" "#F1B556" "#A6E7F3" "#7172AD"])
+  []
+  (get-whitelabeled-colors ["#509EE3" "#88BF4D" "#A989C5" "#EF8C8C" "#F9D45C" "#F2A86F" "#98D9D9" "#7172AD"]))
```

Comment:
The fact that you had to try to eyeball it to figure out what those colors meant is a sign that defining them as a vector is not the way to go. It's definitely weird to have a vector like that and just assume that somebody somewhere knows what they map to. Can you change it to a map like this?

```clj
[:brand :accent1 :accent2 :accent3 :accent4 :accent5 :accent6 :accent7]
```

Can you change this to a map like it is in the FE?

e.g.

```clj
{:brand "#509EE3" 
 :accent1  "#88BF4D"
...
}
```

Reply:
You'll have to change the implementation for `whitelabeled-colors` but it should be simpler